kits/arm/ex_mobile_device_control_v4.py

```diff
@@ -0,0 +1,320 @@
+import hebi
```

Comment:
What is up with the `ex_mobile_device_control_vX.py` scheme here? It sounds like these were iterations to get the example working... If this is the case, we need to delete all but the one we want to keep